### PR TITLE
[Timezones] Adds a toggle for displaying timezone next to the message

### PR DESCRIPTION
**Pull request by RembiTime**

### CODE REVIEW

File: Timezones/Timezones.plugin.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * @name Timezones
  * @author TheCommieAxolotl#0001
  * @description Allows you to display other Users' local times.
- * @version 1.0.0
+ * @version 1.0.1
```
Comment (TheCommieAxolotl) [inline Timezones/Timezones.plugin.js:5]: Please don't bump the version in a pull request.

File: Timezones/Timezones.plugin.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = (() => {
                 },
             ],
             github_raw: "https://raw.githubusercontent.com/TheCommieAxolotl/BetterDiscord-Stuff/main/Timezones/Timezones.plugin.js",
-            version: "1.0.0",
+            version: "1.0.1",
```
Comment (TheCommieAxolotl) [inline Timezones/Timezones.plugin.js:24]: same as above

File: Timezones/Timezones.plugin.js [hunk 3]
```diff
@@ -31,6 +31,12 @@ module.exports = (() => {
                 name: "24 Hour Time",
                 value: false,
             },
+            {
+                type: "switch",
+                id: "showInMessage",
+                name: "Show timezone next to each message",
```
Comment (TheCommieAxolotl) [inline Timezones/Timezones.plugin.js:37]: should be 
```js
name: "Show timezone next to message timestamp",
```

**Review by TheCommieAxolotl — CHANGES_REQUESTED** [review]

**Review by RembiTime — COMMENTED** [review]
Accepted changes in version

**Review by TheCommieAxolotl — APPROVED** [review]

**Pull request merged**